index.spec.js

```diff
@@ -105,7 +105,7 @@ describe('Cypress dotenv plugin', () => {
 
   describe('Optional all argument', () => {
     it('Should return all available env vars', () => {
-      const dotenvConfig = { path: './.env' }
+      const dotenvConfig = { path: '.' }
```

Comment:
I don't really understand why you had to set this config. With `dotenv-flow`, the `path` option points to where the `.env` files are located. Because there are no `.env` folders, it won't work with `{ path: './.env' }`. I think it's better to remove this config entirely.

Reply:
this is a unit test. it is testing that the behavior remains the same if the optional arguments are passed in (this in case, specifying what file to use as the `env` file).
This should not be removed because which file to use is something that is configurable, so we should test for that.